### DISCUSSION TITLE
Admin users, FE users, roles?

### DISCUSSION THREAD

**Discussion by borg** [keystonejs/keystone#6578] [Event Q&A Q3 2021]
What is the suggested way of managing user types and roles, especially considering there currently seems to be only one supported authentication type? In many cases FE users have no business accessing admin, do they still need to be part of same one user table?

**Comment by raveling** [comment]
@borg thanks for your question
We recently answered this in the community Q&A event:
https://www.youtube.com/watch?v=r1IJh-iHm1c&t=2690s

**Reply by IRediTOTO** [reply]
=.= where is the video ?

**Reply by raveling** [reply]
@IRediTOTO  so sorry for the confusion – we had an unfortunate situation with our YouTube account and lost that video from the web 🤦

We intend to fix this and update the timestamp links.
